packages/kuma-gui/src/app/x/components/x-tabs/XTabs.vue

```diff
@@ -67,7 +67,10 @@ const rewrite = () => {
   const $tabindexed = $el.querySelectorAll('[tabindex="0"]')
   const $id = $el.querySelectorAll('[id]')
 
-  $tablist && ['role', 'aria-label'].forEach(attr => $tablist.removeAttribute(attr))
+  if($tablist) {
+    ['role', 'aria-label'].forEach(attr => $tablist.removeAttribute(attr))
+  }
+  
```

Comment:
Same as above, looks like:

```
'@typescript-eslint/no-unused-expressions': ['error', {
  allowTernary: true,
  allowShortCircuit: true
}],
```

would mean we wouldn't have to change non-config code in this PR.